src/js/brandColors/brandColors.test.ts

```diff
@@ -0,0 +1,410 @@
+/* eslint-disable @typescript-eslint/no-var-requires, @typescript-eslint/no-require-imports */
+import { brandColors } from './brandColors';
+
+const designTokens = require('../../figma/tokens.json');
+
+describe('Brand Color', () => {
+  describe('White', () => {
+    it('js tokens for white000 matches figma tokens white000', () => {
```

Comment:
These tests check the export color values match the hex values of the colors in Figma json